### PR TITLE
Add default application templates

### PR DESCRIPTION
**Pull request by DilshanSenarath**
## Related Issues

- https://github.com/wso2/product-is/issues/20107

### CODE REVIEW

File: applications/m2m-application/info.json
Comment (pavinduLakshan) [inline applications/m2m-application/info.json:1]: indentations in this file seem to be different with the rest of the files. is that intentional?

better if we can preserve the formatting of custom-application/info.json

Comment (DilshanSenarath) [reply applications/m2m-application/info.json:1]: Yes I thought the same thing, but when I checked these files n vs code, there is no indentation difference

Comment (pavinduLakshan) [reply applications/m2m-application/info.json:1]: I could see the tab characters when I opened the file in edit mode in github web, and replaced some of these with 4 spaces. Could you check whether you can do the same maybe?

Comment (DilshanSenarath) [reply applications/m2m-application/info.json:1]: https://github.com/wso2-extensions/identity-integration-ui-templates/pull/2/commits/46a211b0ed9bd03a7fdfeafdeaf67760946e67ed

File: applications/custom-protocol-application/info.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+    "name": "Custom Application",
+    "description": "Applications built using niche, legacy or custom protocols.",
+    "image": "${CONSOLE_BASE_URL}/resources/applications/assets/images/illustrations/custom-template.svg",
+    "displayOrder": 5,
+    "category": "DEFAULT",
+    "tags": [
+        "Default"
+    ],
+    "type": "applications"
+}
```
Comment (pavinduLakshan) [inline applications/custom-protocol-application/info.json:12]: Let's add newline.
```suggestion
}

```

File: applications/m2m-application/info.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+                    "logo": "${CONSOLE_BASE_URL}/resources/applications/assets/images/technologies/oauth2.png"
+                }
+            ]
+        },
+        {
+            "key": "featureStatus",
+            "value": "new"
+        }
+    ]
+}
```
Comment (pavinduLakshan) [inline applications/m2m-application/info.json:28]:
```suggestion
}

```

File: applications/mobile-application/info.json [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+                    "logo": "${CONSOLE_BASE_URL}/resources/applications/assets/images/technologies/ios-logo.svg"
+                },
+                {
+                    "displayName": "Flutter",
+                    "logo": "${CONSOLE_BASE_URL}/resources/applications/assets/images/technologies/flutter-logo.svg"
+                }
+            ]
+        }
+    ]
+}
```
Comment (pavinduLakshan) [inline applications/mobile-application/info.json:32]:
```suggestion
}

```

File: applications/single-page-application/info.json [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+                    "logo": "${CONSOLE_BASE_URL}/resources/applications/assets/images/technologies/vue-logo.svg"
+                },
+                {
+                    "displayName": "Javascript",
+                    "logo": "${CONSOLE_BASE_URL}/resources/applications/assets/images/technologies/javascript-logo.svg"
+                }
+            ]
+        }
+    ]
+}
```
Comment (pavinduLakshan) [inline applications/single-page-application/info.json:36]:
```suggestion
}

```

File: applications/traditional-web-application/info.json [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+                    "logo": "${CONSOLE_BASE_URL}/resources/applications/assets/images/technologies/nodejs-logo.svg"
+                },
+                {
+                    "displayName": "PHP",
+                    "logo": "${CONSOLE_BASE_URL}/resources/applications/assets/images/technologies/php-logo.svg"
+                }
+            ]
+        }
+    ]
+}
```
Comment (pavinduLakshan) [inline applications/traditional-web-application/info.json:37]:
```suggestion
}

```

File: applications/custom-application/info.json [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+                    "logo": "${CONSOLE_BASE_URL}/resources/applications/assets/images/technologies/saml.png"
+                },
+                {
+                    "displayName": "WS-Federation",
+                    "logo": "${CONSOLE_BASE_URL}/resources/applications/assets/images/technologies/ws-fed.png"
+                }
+            ]
+        }
+    ]
+}
```
Comment (pavinduLakshan) [inline applications/custom-application/info.json:34]:
```suggestion
}

```

**Review by pavinduLakshan — DISMISSED** [review]

**Review by pavinduLakshan — APPROVED** [review]

**Pull request merged**